errai-common-apt/pom.xml

```diff
@@ -37,8 +37,8 @@
 
         <dependency>
             <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-simple</artifactId>
-            <scope>test</scope>
+            <artifactId>slf4j-log4j12</artifactId>
+            <scope>provided</scope>
```

Comment:
Same as before: why is this changed from test scoped?

Reply:
Same answer as above.